frontend_lib/src/helper.js

```diff
@@ -593,5 +593,34 @@ export const sortWorkspaceList = (workspaceList, lang) => {
   })
 }
 
+export const scrollIntoViewIfNeeded = (elementToScrollTo, fixedContainer) => {
```

Comment:
Just for curiosity, why just not use scrollIntoView?

Reply:
Because it makes the sidebar annoyingly and uselessly jumps when clicking on an item on it.